specificatie/openapi.yaml

```diff
@@ -695,6 +699,19 @@ components:
           type: number
       explode: false
 
+    pandIdentificaties:
+      description: "Een lijst met identificaties van panden uit de BAG. Deze is 16 cijfers lang."
+      name: pandIdentificaties
+      in: query
+      required: false
+      schema:
+        type: array
+        items:
+          type: string
+          pattern: '^[0-9]{16}$'
+          minItems: 1
+          example: '0226100000008856'
```

Comment:
Wordt hier een voorbeeld gegeven voor één item in de array of willen we hier een voorbeeld geven voor de array? Deze constructie zorgt er nl. in swagger-ui voor dat er geen pandidentificaties voorbeeld wordt getoond.

![image](https://user-images.githubusercontent.com/18151729/124725278-07997b00-df0d-11eb-9f70-2f0213e9417b.png)

Als het als volgt wordt gespecificeerd

``` yaml
        items:
          type: string
          pattern: '^[0-9]{16}$'
          minItems: 1
        example:
        - '0226100000008856'
```

dan krijgt je het volgende:

![image](https://user-images.githubusercontent.com/18151729/124725740-67902180-df0d-11eb-8cdf-8ed3e03eca61.png)

<kbd>Try it out</kbd> werkt in beide manieren hetzelfde.